client/components/mma/accountoverview/AccountOverview.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import { rest } from 'msw';
-import { ReactRouterDecorator } from '../../../../.storybook/ReactRouterDecorator';
-import { featureSwitches } from '../../../../shared/featureSwitches';
+import { http, HttpResponse } from 'msw';
+import { ReactRouterDecorator } from '@/.storybook/ReactRouterDecorator';
```

Comment:
thanks for doing this ... we could do with tidying up the import statements throughout the codebase at some point

Reply:
Yeah, I'm afraid I didn't do it very systematically here. Probably best to have another PR where we do this?